yarn-project/pxe/src/storage/private_event_data_provider/private_event_data_provider.test.ts

```diff
@@ -19,8 +19,7 @@ describe('PrivateEventDataProvider', () => {
   let blockNumber: number;
   let eventSelector: EventSelector;
   let txHash: TxHash;
```

Comment:
AI one-shotted updates needed in this test. For these low-complexity refactors it's great.